reverse_proxy/defaults/main.yml

```diff
@@ -6,6 +6,13 @@ zone: tre.example.org
 cran_mirror: https://cran.ma.imperial.ac.uk/
 cran_regex: ^/(bin/windows|src)/contrib/(4\.1|4\.3|4\.4|3\.6)*/*[a-zA-Z0-9_\-\.]*(\.tar\.gz|\.zip|PACKAGES)$
 
+# bioconductor
+bioconductor_mirror: https://bioconductor.statistik.tu-dortmund.de/
+bioc_repo_regex: ^\/packages\/[0-9\.]+/(src|bin\/windows)\/contrib\/[a-zA-Z0-9_\-\.]*(\.tar\.gz|\.zip)$
+bioc_index_regex: ^\/packages\/[0-9\.]+/workflows/src/contrib/PACKAGES$
+bioc_data_regex: ^\/packages/[0-9\.]+/(workflows|data\/(annotation|experiment))\/src\/contrib\/PACKAGES$
```

Comment:
I just ran
```R
if (!require("BiocManager", quietly = TRUE))
    install.packages("BiocManager")

BiocManager::install("BSgenome.Hsapiens.UCSC.hg38")
```
in a local container...
```
...
trying URL 'https://bioconductor.org/packages/3.18/data/annotation/src/contrib/BSgenome.Hsapiens.UCSC.hg38_1.4.5.tar.gz'
Content type 'application/gzip' length 731245539 bytes (697.4 MB)
```
It seems to have failed for other reasons, but it sounds like it could be a whole genome!

Reply:
Actually the regex only allows access to `/PACKAGES` so it should be fine....